packages/frontend/src/components/common/modal/ChartDuplicateModal.tsx

```diff
@@ -0,0 +1,107 @@
+import { SavedChart } from '@lightdash/common';
+import {
+    Button,
+    Group,
+    Modal,
+    ModalProps,
+    Stack,
+    Textarea,
+    TextInput,
+    Title,
+} from '@mantine/core';
+import { useForm } from '@mantine/form';
+import { FC, useEffect } from 'react';
+import {
+    useDuplicateChartMutation,
+    useSavedQuery,
+} from '../../../hooks/useSavedQuery';
+
+interface ChartUpdateModalProps extends ModalProps {
+    uuid: string;
+    onConfirm?: () => void;
+}
+
+type FormState = Pick<SavedChart, 'name' | 'description'>;
+
+const ChartUpdateModal: FC<ChartUpdateModalProps> = ({
+    uuid,
+    onConfirm,
+    ...modalProps
+}) => {
+    const { mutateAsync: duplicateChart, isLoading } =
+        useDuplicateChartMutation({
+            showRedirectButton: true,
+        });
+    const { data: chart, isInitialLoading } = useSavedQuery({ id: uuid });
+
+    const form = useForm<FormState>({
+        initialValues: {
+            name: '',
+            description: '',
+        },
+    });
+
+    const { setValues } = form;
+
+    useEffect(() => {
+        if (!chart) return;
+        setValues({
+            name: 'Copy - ' + chart.name,
+            description: chart.description,
+        });
+    }, [chart, setValues]);
```

Comment:
mantine has a way to set initial values - described here: `https://mantine.dev/form/use-form/#values`

Reply:
I wrote it like this because the update modal used the same mechanism too. See: https://github.com/lightdash/lightdash/blob/main/packages/frontend/src/components/common/modal/ChartUpdateModal.tsx#L47
Do you want me to refactor it here on the duplicate modal?

Reply:
I've updated it to use `setInitialValues` followed by `setValues` just like the other occurences.